python/tk_multi_publish2/api/plugins/collector_instance.py

```diff
@@ -117,6 +117,28 @@ def run_create_properties_widget(self, parent, items):
                 extra=_get_error_extra_info(error_msg)
             )
 
+    def run_on_properties_changed(self, items):
```

Comment:
This method should match the signature of `run_on_context_changed()` as its usage and functionality are very similar.